README.md

```diff
@@ -86,6 +86,7 @@ All methods are intentionally public, but the most readily available are the fol
  * `getDuration()` – Returns the duration of an audio clip in seconds.
  * `isPlaying()` – Returns true if currently playing, false otherwise.
  * `load(url)` – Loads audio from URL via XHR. Returns XHR object.
+ * `cancelLoad(xhrObject)` - Cancels the loading of audio from a URL. Takes the XHR object returned from `load`. Used for early navigation if loading is incomplete.
```

Comment:
maybe...
if im going abort the request and not use the audio, i dont really feel
like storing the rest of it in memory. might as well just instantiate a new
one if that track ever comes up again.

On Fri, Feb 26, 2016 at 12:09 PM, katspaugh <notifications@github.com>
wrote:

> In README.md
> <https://github.com/katspaugh/wavesurfer.js/pull/659#discussion_r54271154>
> :
>
> > @@ -86,6 +86,7 @@ All methods are intentionally public, but the most readily available are the fol
> >   * `getDuration()` – Returns the duration of an audio clip in seconds.
> >   * `isPlaying()` – Returns true if currently playing, false otherwise.
> >   * `load(url)` – Loads audio from URL via XHR. Returns XHR object.
> > + * `cancelLoad(xhrObject)` - Cancels the loading of audio from a URL. Takes the XHR object returned from `load`. Used for early navigation if loading is incomplete.
>
> It's your PR, isn't it? :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/katspaugh/wavesurfer.js/pull/659/files#r54271154>.
>